src/types/onyx/SearchResults.ts

```diff
@@ -0,0 +1,40 @@
+import type {Receipt} from './Transaction';
+
+type SearchResultsInfo = {
+    offset: number;
+    type: string;
+    hasMoreResults: boolean;
+};
+
+type SearchTransaction = {
+    transactionID: string;
+    parentTransactionID?: string;
+    receipt?: Receipt;
+    hasEReceipt?: boolean;
+    created: string;
+    merchant: string;
+    modifiedCreated?: string;
+    modifiedMerchant?: string;
+    description: string;
+    from: {displayName: string; avatarURL: string};
+    to: {displayName: string; avatarURL: string};
```

Comment:
Just noting that this changed in the doc. We're now passing `accountID` and `managerID`, so we'll have to get this information from the `personalDetailsList` returned in the `snapshot_<hash>.data.personalDetailsList` Onyx key